app/Services/BannerFinder.php

```diff
@@ -104,16 +104,16 @@ public static function getBestBanners(array $zones, array $keywords)
         $banners = [];
         foreach ($bannerIds as $bannerId) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$bannerId" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1481)